src/styles/search-container.css

```diff
@@ -0,0 +1,15 @@
+.search-container {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-container_big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-container_small {
+  width: 100%;
+  height: 42px;
```

Comment:
Don't fix container size (if there is no such a requirement). As with the previous comment, avoid using a fixed height unless it's a specific requirement. Content should dictate the height of the container to prevent overflow issues.